crates/runestick/Cargo.toml

```diff
@@ -29,6 +29,8 @@ futures = "0.3.5"
 anyhow = "1.0.32"
 pin-project = "0.4.23"
 
+runestick-macros = {version = "0.6.16", path = "../runestick-macros"}
```

Comment:
Will queue it up for consideration before the next release, but this also changes how `Any` plumbing is derived internally so that it's consistent (through `__internal_impl_external`).